java/pom.xml

```diff
@@ -74,7 +74,7 @@
         <commons.io.version>2.11.0</commons.io.version>
         <commons.text.version>1.10.0</commons.text.version>
         <commons.lang3.version>3.12.0</commons.lang3.version>
-        <debezium.version>2.4.2.Final</debezium.version>
+        <debezium.version>2.6.2.Final</debezium.version>
```

Comment:
could you also build a image of this branch and trigger the integration test pipeline?

Reply:
Passed. https://buildkite.com/risingwavelabs/integration-tests/builds/859